DigitalLearningSolutions.Web/Views/Frameworks/Developer/SignpostingSetStatus.cshtml

```diff
@@ -0,0 +1,74 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks;
+@model CompetencyLearningResourceSignpostingParametersViewModel
+@{
+    var addOrEdit = Model.AssessmentQuestionParameter.IsNew ? "Add" : "Edit";
+    ViewData["Title"] = $"{addOrEdit} Signposting Parameter";
+    ViewData["Application"] = "Framework Service";
+}
+@section NavMenuItems {
+  <partial name="Shared/_NavMenuItems" />
+}
+@section NavBreadcrumbs {
+  <nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+    <div class="nhsuk-width-container">
+      <ol class="nhsuk-breadcrumb__list">
+        <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link trigger-loader" asp-action="ViewFrameworks" asp-route-tabname="Mine">Frameworks</a></li>
+        <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link trigger-loader" asp-action="ViewFramework" asp-fragment="fc-@Model.FrameworkCompetencyId" asp-route-frameworkId="@Model.FrameworkId" asp-route-tabname="Structure">Framework Structure</a></li>
+        <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link trigger-loader" asp-action="EditCompetencyLearningResources" asp-route-frameworkId="@Model.FrameworkId" asp-route-frameworkCompetencyId="@Model.FrameworkCompetencyId" asp-route-frameworkCompetencyGroupId="@Model.FrameworkCompetencyGroupId">Signposting</a></li>
+        <li class="nhsuk-breadcrumb__item">Parameters</li>
+      </ol>
+      <p class="nhsuk-breadcrumb__back"><a class="nhsuk-breadcrumb__backlink" asp-action="EditCompetencyLearningResources" asp-route-frameworkId="@Model.FrameworkId" asp-route-frameworkCompetencyId="@Model.FrameworkCompetencyId" asp-route-frameworkCompetencyGroupId="@Model.FrameworkCompetencyGroupId">Back to Signposting</a></p>
+    </div>
+  </nav>
+}
+<link rel="stylesheet" href="@Url.Content("~/css/frameworks/frameworksShared.css")" asp-append-version="true">
+<partial name="Developer/_SignpostingParametersHeader" model="Model" />
+
+<h2>Signposting status</h2>
+<label class="nhsuk-label nhsuk-u-margin-bottom-4">
+  Choose whether the learning resorice should be tagged as Recommended or Essential. Only choose Essential if completion of the learning is a mandatory requirement for the role.
```

Comment:
Reword the opening sentence to "Choose whether the learning resource should be tagged as Essential."